loadflow/loadflow-api/src/main/java/com/powsybl/loadflow/LoadFlowParameters.java

```diff
@@ -58,6 +66,10 @@
     public static final boolean DEFAULT_SIMUL_SHUNT = false;
     public static final boolean DEFAULT_READ_SLACK_BUS = false;
     public static final boolean DEFAULT_WRITE_SLACK_BUS = false;
+    public static final boolean DEFAULT_VOLTAGE_REMOTE_CONTROL = false;
```

Comment:
I wonder if the default value for voltage remote control could be true. @geofjamg what is your opinion about that ? Do you understand here also remote control for tap changers and swicthes ? Or just for generators and static var compensators ?